dts/bindings/serial/uart-controller.yaml

```diff
@@ -23,3 +23,22 @@ properties:
       - "none"
       - "odd"
       - "even"
+  stop-bits:
+    type: string
+    description: |
+      Sets the number of stop bits.
+    enum:
+      - "0_5"
+      - "1"
+      - "1_5"
+      - "2"
+  data-bits:
+    type: string
```

Comment:
Why is this a string?

Reply:
I guess it was to be consistent with `stop-bits`, but it doesn't have to be. Changed to `int`. Also did a rebase.